app/assets/javascripts/views/feedbacks.js

```diff
@@ -3,18 +3,42 @@ ReviewSite = ReviewSite || {};
 var activateAccordion = function() {
   $("#accordion").accordion({
     heightStyle:"content",
-    active: 4,
-    collapsible: true
+    active: 0,
+    collapsible: false,
+    create: function(event, ui) {
+      toggleFields({ commentsHeader: ui.header });
+    }
   });
 
   $("#accordion").on("accordionactivate", function (event, ui) {
+    toggleFields({ oldHeader: ui.oldHeader, newHeader: ui.newHeader });
+
     if (ui.newPanel.length) {
       $.scrollTo(ui.newHeader, {offset:{top:-100}});
     }
   });
+};
+
+var toggleFields = function(headers) {
+  for(var key in headers) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.